Doc/library/asyncio-eventloop.rst

```diff
@@ -1723,6 +1723,14 @@ on Unix and :class:`ProactorEventLoop` on Windows.
       `MSDN documentation on I/O Completion Ports
       <https://docs.microsoft.com/en-ca/windows/desktop/FileIO/i-o-completion-ports>`_.
 
+.. class:: EventLoop
+
+    Uses the most efficient available event loop for the given
+    platform.
```

Comment:
```suggestion
    Uses the most efficient available event loop for the given
    platform.
    
    It is an alias for  :class:`SelectorEventLoop` on Unix and :class:`ProactorEventLoop` on Windows.
```

Reply:
Not quite. I don't like any sentence describing a class object starting with "Uses". Classes are nouns, and the description should explain what the class *is*, not what it *does*. Also, the current algorithm for choosing a default is now specified in two places. I think it should be specified only here (and the earlier text should be changed to claim that `EventLoop` is the class used by default).

It's also a good idea to clarify that both concrete classes are subclasses of `AbstractEventLoop` (but if you're tired of going around in review you can skip that).

Reply:
ok I've pushed a change with some doc tweaks